src/main/java/net/minecraftforge/common/BiomeDictionary.java

```diff
@@ -249,251 +214,231 @@ public static boolean areBiomesEquivalent(Biome biomeA, Biome biomeB)
     }
 
     /**
-     * Checks to see if the given biome is registered as being a specific type
+     * Checks if the given type has been added to the given biome.
      *
-     * @param biome the biome to be considered
-     * @param type the type to check for
-     * @return returns true if the biome is registered as being of type type, false otherwise
      */
-    public static boolean isBiomeOfType(Biome biome, Type type)
+    public static boolean hasType(Biome biome, Type type)
     {
-        checkRegistration(biome);
-        return containsType(getBiomeInfo(biome), type);
+        return getTypes(biome).contains(type);
     }
 
     /**
-     * Checks to see if the given biome has been registered as being of any type
-     * @param biome the biome to consider
-     * @return returns true if the biome has been registered, false otherwise
-     */
-    public static boolean isBiomeRegistered(Biome biome)
-    {
-        return biomeInfoMap.containsKey(Biome.REGISTRY.getNameForObject(biome));
-    }
-
-    public static void registerAllBiomes()
-    {
-        FMLLog.warning("Redundant call to BiomeDictionary.registerAllBiomes ignored");
-    }
-    /**
-     * Loops through the biome list and automatically adds tags to any biome that does not have any
-     * This is called by Forge at postinit time. It will additionally dispatch any deferred decorator
-     * creation events.
+     * Checks if any types have been added to the given biome.
      *
-     * DO NOT call this during world generation
      */
-    public static void registerAllBiomesAndGenerateEvents()
+    public static boolean hasTypes(Biome biome)
     {
```

Comment:
I feel like this name is rather confusing given that hasType accepts a Type argument, thus implying that hasTypes should therefore be used to check whether a biome is assigned multiple types as provided by the modder.